### ISSUE TITLE
Dart Command Line Too Slow

### ISSUE THREAD

**Issue by putraxor** [Dart-Code/Dart-Code#1054]
It seems run dart command line app too slow compared to manually `dart script.dart` on terminal, this is my config:
```javascript
        {
            "request": "launch",
            "type": "dart",
            "name": "Console",
            "program": "${workspaceFolder}/lib/console.dart"
        }
```
**Dart Plugin v2.15.0**

**Comment by DanTup** [comment]
Can you give a rough idea of what "too slow" means? 1 second more? 10 seconds? 30 seconds? How are you measuring - by output to the Debug Console, or something else?

Can you try capturing a log (run the `Dart: Capture Logs` command, then run your app, then click the `Stop Logging` button once you're done) to see if the timestamps in there show any indication of where the time was spent.

**Comment by putraxor** [comment]
[Dart-Code-Log.txt](https://github.com/Dart-Code/Dart-Code/files/2159701/Dart-Code-Log.txt)

Hi DanTup, this is the log file

**Comment by DanTup** [comment]
Yeah, this is a valid way to run. I misunderstood and thought you were running a Dart CLI app though - you're actually running Flutter tests?

Flutter Test is kinda slow for me too - looks like it takes about 5 seconds to start up:

```
[21:10:20 GMT+0700 (WIB)] [FlutterTest] <== {"test":{"id":1,"name":"loading /Users/wizard/Desktop/learn_flutter/test/console.dart","suiteID":0,"groupIDs":[],"metadata":{"skip":false,"skipReason":null},"line":null,"column":null,"url":null},"type":"testStart","time":1}
[21:10:25 GMT+0700 (WIB)] [FlutterTest] <== [{"event":"test.startedProcess","params":{"observatoryUri":"http://127.0.0.1:50408/"}}]
```

I'll do some digging and see if I can find out why it's so slow. Is it faster if you run with Debug -> Start Without Debugging?
